app-misc/prog-express/prog-express-3.7.8.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils multilib
```

Comment:
I do not see any reason to use `eutils.eclass` here as well, please use `desctop.eclass`, it provides functionality to handle menus and icons :)

Reply:
Ah, I see, will be changed.